cdap-ui/app/cdap/components/AbstractWidget/DLPCustomWidget/index.tsx

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import * as React from 'react';
+import DLPRow, {
+  ITransformProp,
+  FilterOption,
+} from 'components/AbstractWidget/DLPCustomWidget/DLPRow';
+import ThemeWrapper from 'components/ThemeWrapper';
+import AbstractMultiRowWidget, {
+  IMultiRowProps,
+} from 'components/AbstractWidget/AbstractMultiRowWidget';
+import { objectQuery } from 'services/helpers';
+import { WIDGET_PROPTYPES } from 'components/AbstractWidget/constants';
+
+interface IDLPWidgetProps {
+  transforms: ITransformProp[];
+  filters: FilterOption[];
+  delimiter?: string;
+}
+
+interface IDLPProps extends IMultiRowProps<IDLPWidgetProps> {}
+
+class DLPWidgetView extends AbstractMultiRowWidget<IDLPProps> {
+  public editRow = (id, value) => {
```

Comment:
this method seems to be the same. Is there a need to overwite it?

Reply:
Removed